app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -422,6 +423,14 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                         getStringResForStatus(txDetails.txStatus, needsYourConfirmation),
                         getColorForStatus(txDetails.txStatus)
                     )
+                    if (needsYourConfirmation) {
```

Comment:
we have check for this at the top outside of the view stub layout, right? As button container is not part of the view stub I don't think this is the right place to set visibility of the buttons.

Reply:
The rejections have different rules on whether to display the button or not.

Reply:
sure, but can we do this check at the top in one place. Otherwise we are doing this work twice

Reply:
Done in 2720218293f4bb404309d24d030988c9373f9e6e